src/PKSimRDependencyResolution/PKSimRDependencyResolution.csproj

```diff
@@ -0,0 +1,72 @@
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <PlatformTarget>x64</PlatformTarget>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\..\SolutionInfo.cs" Link="Properties\SolutionInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\PKSim.R\PKSim.R.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="filelist.txt">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Autofac" Version="4.9.4" />
+    <PackageReference Include="csmpfit" Version="1.1.1" />
+    <PackageReference Include="DevExpress.Data" Version="21.2.3" />
+    <PackageReference Include="LumenWorksCsvReader" Version="4.0.0" />
+    <PackageReference Include="MathNet.Numerics" Version="4.15.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.1.0" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Options" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Extensions.Primitives" Version="3.1.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    <PackageReference Include="NPOI" Version="2.5.5" />
+    <PackageReference Include="OSPSuite.Assets" Version="12.0.296" />
+    <PackageReference Include="OSPSuite.Assets.Images" Version="12.0.296" />
+    <PackageReference Include="OSPSuite.Core" Version="12.0.296" />
+    <PackageReference Include="OSPSuite.FuncParser" Version="4.0.0.54" />
+    <PackageReference Include="OSPSuite.Infrastructure" Version="12.0.296" />
+    <PackageReference Include="OSPSuite.Infrastructure.Autofac" Version="12.0.296" />
+    <PackageReference Include="OSPSuite.Infrastructure.Import" Version="12.0.296" />
+    <PackageReference Include="OSPSuite.R" Version="12.0.296" />
+    <PackageReference Include="OSPSuite.Serializer" Version="3.0.0.1" />
+    <PackageReference Include="OSPSuite.SimModel" Version="4.0.0.56" />
+    <PackageReference Include="OSPSuite.Utility" Version="4.1.0.5" />
+    <PackageReference Include="Portable.BouncyCastle" Version="1.8.9" />
+    <PackageReference Include="Serilog" Version="2.10.0" />
```

Comment:
Are all of these required or can we somehow reference InfrasturacutreImport that depends on these already? Why is this package depending on so many DIRECTLY?